### PR TITLE
Add RBAC configuration to example

### PR DESCRIPTION
**Pull request by rfancn**
This commit fixed the issue: [Need add k8s RBAC configuration for plugin to work correctly](https://github.com/rabbitmq/rabbitmq-peer-discovery-k8s/issues/15)

#### CHANGES
- add RBAC related role/rolebinding/serviceaccount
- add serviceaccountName to rabbitmq_statefulset.yaml
- change README to mention apply rabbitmq_rbac.yaml before applying rabbitmq_statefulset.yaml

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@rfancn Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/rabbitmq?repositoryId=rabbitmq/rabbitmq-peer-discovery-k8s&pullRequestId=16)!

[Click here](https://cla.pivotal.io/sync/rabbitmq?repositoryId=rabbitmq/rabbitmq-peer-discovery-k8s&pullRequestId=16) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by pivotal-issuemaster** [comment]
@rfancn Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/rabbitmq?repositoryId=rabbitmq/rabbitmq-peer-discovery-k8s&pullRequestId=16)!

**Comment by michaelklishin** [comment]
@rfancn thank you!

@Gsantomaggio WDYT?

**Comment by Gsantomaggio** [comment]
@rfancn Thanks
@michaelklishin It is ok, going to merge it

File: examples/k8s_statefulsets/README.md [hunk 1]
```diff
@@ -23,7 +23,13 @@ $ minikube start --cpus=2 --memory=2040 --vm-driver=virtualbox
 $ kubectl create namespace test-rabbitmq
 ```
 
-* Deploy the  `YAML` file:
+* Deploy RBAC `YAML` file:
+
+```
+$ kubectl create -f examples/k8s_statefulsets/rabbitmq_rbac.yaml
+```
```
Comment (Gsantomaggio) [inline examples/k8s_statefulsets/README.md]: Can you please add a note here? 
why we need `rbac`,  something like you already wrote here:
```
For kubernetes 1.6 or above, it enabled RBAC by default
```

Comment (rfancn) [reply examples/k8s_statefulsets/README.md]: added as you suggested.

**Comment by Gsantomaggio** [comment]
@rfancn Thank you

**Pull request merged**